### ISSUE TITLE
Remove render step in interactions panel

### ISSUE THREAD

**Issue by vanessayuenn** [storybookjs/storybook#31429]
[Check the designs out here](https://www.figma.com/design/MDOrXFR6s1YjxTdEyC0EMQ/Unified-UI-Testing?node-id=4631-26174&t=nFHMuOC6o8hty1ir-1)

**Tasks:**
- Remove render step
- Add error message to Interactions, A11y, and VT tabs when render fails, and blocks those addons from running until errors are resolved

Out of scope:
Any changes to testing widget
